### ISSUE TITLE
Mouse actions on CKEditor scrollbar throws 'p.getAscendant is not a function'

### ISSUE THREAD

**Issue by kdambrauskas** [ckeditor/ckeditor-dev#515]
## Bug

Mouse events on CKEditor scrollbar results in error 'p.getAscendant is not a function'

### Steps to reproduce
1. Make editor scrollable
2. Mouse over scrollbar
3. Profit.

### Expected result
No errors

### Actual result
Errors

## Other details

Clicking the scrollbar also causes an exception.

* Version Chrome 59.0.3071.86 (Official Build) (64-bit)
* OS: Win 10
* CKEditor version: 4.7.0
* Installed CKEditor plugins: Official site http://ckeditor.com/

**Comment by jonira** [comment]
We have experienced same issue. Can be reproduced in http://ckeditor.com/ demos. Using chrome Version 59.0.3071.86 (Official Build) (64-bit)

**Comment by mlewand** [comment]
I can confirm, it's being thrown by `tableselection` plugin in https://github.com/ckeditor/ckeditor-dev/blob/621199c/plugins/tableselection/plugin.js#L265

Here's the stack from dev version:
```
plugin.js:265 Uncaught TypeError: target.getAscendant is not a function
    at CKEDITOR.dom.document.fakeSelectionMouseHandler (plugin.js:265)
    at CKEDITOR.dom.document.listenerFirer (event.js:144)
    at CKEDITOR.dom.document.CKEDITOR.event.fire (event.js:290)
    at HTMLDocument.<anonymous> (domobject.js:46)
```

**Comment by rajeev-khangar** [comment]
I am also getting same issue when over cursor on scrollbar. see attachment below.
![screenshot from 2017-08-04 16-05-54](https://user-images.githubusercontent.com/10833286/28965764-5d179554-7930-11e7-9a8d-771392a08643.png)
![screenshot from 2017-08-04 16-17-17](https://user-images.githubusercontent.com/10833286/28965777-69c589dc-7930-11e7-9ecc-3e80b4a302f0.png)

**Comment by mlewand** [comment]
@rajeev-khangar your first screen is a different issue #527 that is fixed in 4.7.2 which we're about to release.

The second screen was fixed in 4.7.1, please update your editor and you will no longer experience the issue.
